### DISCUSSION TITLE
Migrating to Duende

### DISCUSSION THREAD

**Discussion by hsynlms** [skoruba/IdentityServer4.Admin#903] [General]
Hello @skoruba ,

Let me thank you first in the first place for that product.

I just wonder that does it make sense to migrate from IdentityServer4 to Duende Identity Server? If yes, is it safe to use [Skoruba Duende Admin](https://github.com/skoruba/Duende.IdentityServer.Admin) over the existing Skoruba Admin v1 setup?

**Comment by skoruba** [comment]
Hello @hsynlms 

for sure, It definitely make sense to migrate to Duende IdentityServer.

You can try to use my Admin UI for Duende IdentityServer, please let me know, if you will need some help.

Thanks for reaching out to me. 🙂

**Reply by grantvine-sc** [reply]
Hi Skoruba,

Would the process to do this be the same as detailed on the Duende site (for migrations etc), however replacing your libraries with your corresponding Duende ones ? Or would the better approach to be to provision a new instance/template from your Duende one and then point to a database that has had the migrations performed against it ? Or maybe a combination (upgrade, then replace because your other template differs/is more efficient/maintained etc?)
